internal-log-fetcher/src/trace_consumer.rs

```diff
@@ -44,14 +44,14 @@ impl TraceConsumer {
 
         let stdout_log_file = OpenOptions::new()
             .create(true)
-            .write(true)
```

Comment:
Could you comment on the effect of this change?

Reply:
From `fs.rs`, `OpenOptions::new().append(true)` documentation:

> /// Note that setting `.write(true).append(true)` has the same effect as
> /// setting only `.append(true)`.

And the `write(true)` call was automatically removed by Cargo Clippy.

@georgeee Do you want me to return it back?

Reply:
No, it's fine by me, just wanted to ensure we didn't change code in some unexpected ways